app/views/items/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= form_with model: @item do |form| %>
+<%= form.label :name, "Name : " %>
+<%= form.text_field :name %>
+<%= form.label :quantity, "quantity : "%>
+<%= form.number_field :quantity %>
+<%= form.label :pric, "Price" %>
+<%= form.number_field :price %>
+<%= form.select(:sub_category_id, options_for_select(@subcategories.map do |subcategory|
```

Comment:
use {} instead of do end for single line blocks,
move options_for_select(@subcategories.map do |subcategory| [subcategory.title, subcategory.id] end  to helper

Reply:
optional

Reply:
done